package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "AboutAndNews",
+  "version": "",
+  "description": "",
+  "author": "",
+  "license": "",
+  "engines": {
+    "node": ">=6.13.0"
+  },
+  "scripts": {
+    "start": "nodemon server/index.js",
+    "build": "webpack -d --watch",
+    "seed": "node database/seed.js",
+    "database": "mysql -u root < schema.sql"
+  },
+  "dependencies": {
+    "@babel/core": "^7.2.2",
+    "Faker": "^0.7.2",
+    "babel": "^6.23.0",
+    "babel-loader": "^8.0.4",
+    "body-parser": "^1.18.3",
+    "css-loader": "^2.0.1",
+    "express": "^4.16.4",
+    "faker": "^4.1.0",
+    "mysql": "^2.16.0",
+    "mysql2": "^1.6.4",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "sequelize": "^4.42.0",
+    "style-loader": "^0.23.1",
+    "webpack": "^4.28.0",
+    "webpack-cli": "^3.1.2"
+  },
+  "devDependencies": {
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
```

Comment:
imo probably better to use airbnb's ruleset for `eslint`

Reply:
I'll look into that, thanks Robin!